ui/job-view/details/BugFiler.jsx

```diff
@@ -442,6 +442,7 @@ export class BugFilerClass extends React.Component {
           .filter(prodVer => prodVer.is_active)
           .slice(-1)[0];
         const payload = {
+          type: 'defect',
```

Comment:
Thank you for the patch. This line should be dropped because the bugzilla.py hardcodes the type.

Reply:
Okay, removed!